lib/src/components/NxVulnerabilityDetails/details/DetailLink.tsx

```diff
@@ -9,10 +9,13 @@ import React, {FunctionComponent} from 'react';
 
 import NxTextLink, { Props as NxTextLinkProps } from '../../NxTextLink/NxTextLink';
 
-export type Props = NxTextLinkProps;
+export interface Props extends NxTextLinkProps {
+  // react-commonmark is known to pass this
+  nodeKey?: string | number;
```

Comment:
Fixing this is borderline out of scope, but since the SSR tests check that there are no console errors it had to be fixed. Without these changes the `nodeKey` prop ends up getting passed to the `<a>` JSX element which causes React to complain.